src/components/anchor/nz-anchor.component.ts

```diff
@@ -66,10 +66,25 @@ export class NzAnchorComponent {
     this.registerScrollEvent();
   }
 
+  /**
+   * The offset from top when arrive the window top
+   * 距离窗口顶部达到指定偏移量后触发
+   * @type {number}
+   */
   @Input() nzOffsetTop: number = 0;
 
+  /**
+   * The bounds of anchor
+   * 锚点区域边界
+   * @type {number}
+   */
   @Input() nzBounds: number = 5;
 
+  /**
+   * the event of scroll, when scroll to a anchor

```

Comment:
`The event triggered by scrolling to an anchor`
`a anchor` -> `an anchor`
Please refer to it.